spec/tests/check.fmf

```diff
@@ -14,7 +14,17 @@ description: |
     panic detection, core dump collection or collection of system
     logs.
 
-    To control checks for a given test, use L1 ``check`` key.
+    The following checks are currently available:
+
+    dmesg
+        check the kernel righ buffer and store its contents
+
+    avc
+        check for unexpected SELinux AVC denials, gather details
+        about the ``selinux-policy`` related packages installed on
+        the system
+
+    .. versionadded:: 1.28
```

Comment:
I’d guess this directive should be aligned to the same level as the text above, and we should probably have one for each check.